src/metrics/test_coverage/index.ts

```diff
@@ -0,0 +1,90 @@
+import { logger } from "../../core";
+import { decode } from "js-base64";
+import { keys, includes, find, propEq, toLower} from "ramda";
+import { DataEvent, MetricsSignature } from "../../interfaces";
+import {
+  WorkflowJobTestCoveragePayload,
+  WorkflowJobTestCoverageOutput,
+} from "./interfaces";
+import moment from "moment";
+import octokit from "../../core/octokit";
+import { isWorkflowJob } from "../../signatureConditions";
```

Comment:
- [ ] isWorkflowJob is not used in the code

Reply:
Done